datafusion/core/tests/fuzz_cases/join_fuzz.rs

```diff
@@ -102,6 +103,7 @@ async fn test_inner_join_1k_filtered() {
 }
 
 #[tokio::test]
+#[allow(unused_qualifications)]
```

Comment:
Let's have an issue to clean up this as a follow-up.

Reply:
is this something you can file ?  I suspect it will be a good first issue for someone

Reply:
Issue created: https://github.com/apache/datafusion/issues/13162